L1Trigger/L1TGEM/python/me0TriggerDigis_cff.py

```diff
@@ -7,6 +7,7 @@
 
 me0TriggerRealDigiTask = cms.Task(simMuonME0PadDigis, me0TriggerDigis)
 me0TriggerAllDigiTask = cms.Task(me0TriggerRealDigiTask, me0TriggerPseudoDigiTask)
+ge0TriggerAllDigiTask = cms.Task(me0TriggerRealDigiTask, ge0TriggerPseudoDigiTask)
```

Comment:
Looks like the `me0TriggerRealDigiTask` needs to be adapted to ge0 just like the `pseudoDigiTask`, the call to the `ME0PadDigiProducer` happens inside that task.